frontends/verific/verific.cc

```diff
@@ -1153,6 +1153,26 @@ void VerificImporter::import_netlist(RTLIL::Design *design, Netlist *nl, std::se
 	for (auto net : anyseq_nets)
 		module->connect(net_map_at(net), module->Anyseq(new_verific_id(net)));
 
+	char *id_name;
+	TypeRange *type_range;
+	FOREACH_MAP_ITEM(nl->GetTypeRangeTable(), mi, &id_name, &type_range)
+	{
+		if (!type_range)
+			continue;
+		if (!type_range->IsTypeEnum())
+			continue;
+		auto wire = module->wire(RTLIL::escape_id(id_name));
+		log_assert(wire);
```

Comment:
@cliffordwolf Is this too strict?

Reply:
LGTM. We can still adjust this should we find cases where this triggers.